sdk/servicebus/azure-servicebus/tests/stress_tests/test_stress_queues.py

```diff
@@ -157,9 +157,10 @@ def test_stress_queue_receive_large_batch_size(self, servicebus_namespace_connec
 
     # Cannot be defined at local scope due to pickling into multiproc runner.
     class ReceiverTimeoutStressTestRunner(StressTestRunner):
-        def OnSend(self, state, sent_message):
+        def OnSend(self, state, sent_message, sender):
             '''Called on every successful send'''
             if state.total_sent % 10 == 0:
+                # To make it time out and reconnect
```

Comment:
may I ask what would `time.sleep(self.max_wait_time + 5)` make time out?

is it making the `receive` time out? but AFAIK if receive times out, it won't trigger a reconnection of the receiver.
apart from that the default config for `max_wait_time` is just 5 ( I assume it's in unit of seconds ), wondering how could 5 seconds time out trigger a reconnection.

Reply:
It's making the receive loop time out, as you suspect.  The test is explicitly `test_stress_queue_pull_receive_timeout`, so that's all it was intended to validate, basically sparse message flow beyond the timeout threshold.

There's a test below for close-open that validates the more aggressive "reconnection" case. (Truthfully in the future a full chaos monkey suite would be nice but that seems incremental)

Reply:
yea, after reading the `def _Receive`, I understand how it works.

however, I still think the description is a bit incomplete:
I would put it as `make receive time out, in push mode it would trigger receiver reconnection"

Reply:
```suggestion
                # To make receive time out, in push mode this delay would trigger receiver reconnection
```
(being lazy doing this here and batching it with your other suggestion) :P